lib/when_committed.rb

```diff
@@ -1,30 +1,42 @@
 require 'when_committed/version'
+require 'active_record'
 
-module WhenCommitted
-  module ActiveRecord
-    def self.included(base)
-      base.after_commit :run_when_committed_callbacks
-      base.after_rollback :clear_when_committed_callbacks
+module WhenCommitted::ActiveRecord
+
+  extend ActiveSupport::Concern
```

Comment:
I'm pretty sure active_record depends on active_support, so you don't need to specify it explicitly in the gemspec.
That said, I'm not a fan of `ActiveSupport::Concern`, it doesn't really save you any keystrokes or make the code more readable.